### ISSUE TITLE
Hoe does sober work on linux, when it dls from a apk?

### ISSUE THREAD

**Issue by JumpIn-Git** [vinegarhq/sober#431]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

I know this is not a problem, but im curiuos, how does sober make a apk native to linux? would it be possible to make any apk native?

**Comment by tunis4** [comment]
> how does sober make a apk native to linux?

it loads the binary, implements the android APIs that it needs and does workarounds to make the experience similar to desktop

> would it be possible to make any apk native?

yes but it requires a massive amount of work specific to the app

**Comment by JumpIn-Git** [comment]
wow that is cool, just wondering, what would you need to change to make it for a other app, lets say clash royale

**Comment by MiningTcup** [comment]
[waydroid](https://waydro.id/)

**Comment by SSSSSSSSSSSera** [comment]
> [waydroid](https://waydro.id/)

No.

**Comment by k1yrix** [comment]
> [waydroid](https://waydro.id/)

Sober is NOT an emulator

**Comment by MiningTcup** [comment]
Waydroid is NOT an emulator

**Comment by JumpIn-Git** [comment]
> Waydroid is NOT an emulator. It can run games like Roblox so I'm sure it can do clash royale or whatever silly mobile game you wanna play. Sober isn't Waydroid, and Sober is much better for Roblox, but Waydroid is also the solution for other apps.

it is a emulator...

**Comment by MiningTcup** [comment]
No, it's a "containerized environment." If it was an emulator, why wouldn't it work on x11?

**Comment by MiningTcup** [comment]
also this is funny that we are arguing over this on "Hoe does sober work on linux, when it dls from a apk?"

**Comment by JumpIn-Git** [comment]
> also this is funny that we are arguing over this on "Hoe does sober work on linux, when it dls from a apk?"

wdym, just a typo, dont need to be all professional about a questions title
